raiden-ts/src/path/epics.ts

```diff
@@ -466,6 +463,57 @@ export const pfsCapacityUpdateEpic = (
     ),
   );
 
+/**
+ * When monitoring a channel (either a new channel or a previously monitored one), send a matching
+ * PFSFeeUpdate to path_finding global room, so PFSs can pick us for mediation
+ * TODO: Currently, we always send Zero fees; we should send correct fee data from config
+ *
+ * @param action$ - Observable of channelMonitor actions
+ * @param state$ - Observable of RaidenStates
+ * @param deps - Raiden epic dependencies
+ * @returns Observable of messageGlobalSend actions
+ */
+export const pfsFeeUpdateEpic = (
+  action$: Observable<RaidenAction>,
+  state$: Observable<RaidenState>,
+  { log, address, network, signer, config$ }: RaidenEpicDeps,
+): Observable<messageGlobalSend> =>
```

Comment:
any specific reason for using GlobalSend instead of broadcast in the naming here?

Reply:
Sorry, I don't understand what you mean, can you rephrase?
There're 2 actions for sending messages:
- `messageSend.request` is based on `meta.address` to send to a specific Address (transport will take care of inviting the user, waiting for it to be online and joined before sending, etc)
- `messageGlobalSend` instead takes the name of one of the joined global rooms, and send it to it, without waiting for replies (as we don't even listen on these rooms).

The 2nd one is the one used here, it sends the output message once to a global room (in this case, `raiden_<network>_path_finding`); this is the only way to broadcast a message

Reply:
`global send` means `broadcast`, right? What I am asking, since there is a well-known term aka `broadcast`, is there is any specific reason that the action is not named `messageBroadcast` instead?

Reply:
I am just curious :)

Reply:
Yes, `broadcast` also means `global send` (i.e. send to a global/broadcast room); I just wanted to make a parallel with `messageSend`, since they work almost the same from transport's perspective, simply one sends to an address and the other sends to a global/broadcast room. This was just naming, since the rooms were already named `global` on transport and config.